addon/mixins/regenerated/models/new-platform-flexberry-g-i-s-map-layer.js

```diff
@@ -6,26 +6,52 @@ import Ember from 'ember';
 import DS from 'ember-data';
 import { Projection } from 'ember-flexberry-data';
 
-/**
-  Mixin containing map model attributes, relations & projections.
-
-  @class NewPlatformFlexberyGISMapLayerModelMixin
-  @extends <a href="http://emberjs.com/api/classes/Ember.Mixin.html">Ember.Mixin</a>
-*/
 export let Model = Ember.Mixin.create({
   name: DS.attr('string'),
-  type: DS.attr('string'),
+  description: DS.attr('string'),
+  keyWords: DS.attr('string'),
+
+  /**
+    Non-stored property.
+
+    @property anyText
+  */
+
+  anyText: DS.attr('string'),
```

Comment:
for which purposes you add this property?

Reply:
In .NET Objects assembly it is non-stored computed property for full text search.
This is a generation result, all computations are defined in related model.